test/e2e/servicecatalog/podpreset.go

```diff
@@ -262,6 +280,10 @@ var _ = SIGDescribe("[Feature:PodPreset] PodPreset", func() {
 		if !reflect.DeepEqual(originalPod.Spec.Containers[0].Env, pod.Spec.Containers[0].Env) {
 			framework.Failf("env of pod container does not match the env of the original pod: expected %#v, got: %#v", originalPod.Spec.Containers[0].Env, pod.Spec.Containers[0].Env)
 		}
+		if !reflect.DeepEqual(originalPod.Spec.InitContainers[0].Env, pod.Spec.InitContainers[0].Env) {
+			framework.Failf("env of pod init container does not match the env of the original pod: expected %#v, g    ot: %#v", originalPod.Spec.InitContainers[0].Env, pod.Spec.InitContainers[0].Env)
```

Comment:
unnecessary spaces in `g    ot:`

Reply:
Good catch! That was a result of Copy/paste in vim